pages/index.tsx

```diff
@@ -34,9 +35,34 @@ export default function Home({
   errorCode = null,
   userArrivedFromUioMobile = false,
   urlPrefixes,
+  enableGoogleAnalytics,
 }: HomeProps): ReactElement {
   const { t } = useTranslation('common')
 
+  // Once CSP is enabled, if you change the GA script code, you need to update the hash in csp.js to allow
+  // this script to run. Chrome dev tools will have an error with the correct hash value to use.
+  // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#Unsafe_inline_script
+  const gaScript = `
+        window.dataLayer = window.dataLayer || []
+        function gtag(){dataLayer.push(arguments)}
+        gtag('js', new Date())
+        // For details see: https://support.google.com/analytics/answer/9310895?hl=en
+        // https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization
+        gtag('config', 'UA-3419582-2', { 'anonymize_ip': true }) // www.ca.gov
+        gtag('config', 'UA-3419582-31', { 'anonymize_ip': true }) // edd.ca.gov`
+
+  const googleAnalytics = (
+    <>
+      {/* Global site tag (gtag.js) - Google Analytics */}
+      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-3419582-2" />
```

Comment:
I feel like there's a teeny bit of a bad code smell here with the hardcoded IDs.

Reply:
Hmm. Is there a constants file I could move this to? It doesn't make sense for it to be an env var.